### PR TITLE
various small web site updates

### PR DESCRIPTION
**Pull request by kevinb9n**
I'm not using docusaurus (yet?)... just trying to tweak text in ways that I hope won't upset the programming.

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -81,13 +81,21 @@ const config = {
             title: 'Docs',
             items: [
               {
-                label: 'User Guide',
+                label: 'Draft User Guide',
```
Comment (netdpb) [inline docs/docusaurus.config.js:84]: Isn't everything a draft at this point?

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -81,13 +81,21 @@ const config = {
...
                 to: '/docs/user-guide',
               },
               {
-                label: 'Specification',
+                label: 'Current Javadoc',
```
Comment (netdpb) [inline docs/docusaurus.config.js:88]:
```suggestion
                label: 'Javadoc',
```

File: docs/docusaurus.config.js [hunk 3]
```diff
@@ -81,13 +81,21 @@ const config = {
...
+                to: '/docs/api/org/jspecify/nullness/package-summary.html',
+              },
+              {
+                label: 'Draft Spec',
                 to: '/docs/spec',
               },
+              {
+                label: 'JSpecify Wiki',
```
Comment (netdpb) [inline docs/docusaurus.config.js:96]:
```suggestion
                label: 'Wiki',
```

otherwise "JSpecify" goes everywhere.

**Review by netdpb — APPROVED** [review]

File: docs/docusaurus.config.js (hunk 1 above)
Comment (kevinb9n) [reply docs/docusaurus.config.js:84]: Yes; I think it probably benefits from being labeled as such as much as possible.

**Comment by kevinb9n** [comment]
Merged. Javadoc link is right but not working; I think the f/w expects it to be an "internal" link and I have to make it like the github link.

**Pull request merged**